CMakeLists.txt

```diff
@@ -900,7 +900,7 @@ include(ExternalProject)
 
 # ---[ Dependencies ---[ FBGEMM doesn't work on x86 32bit and
 # CMAKE_SYSTEM_PROCESSOR thinks its 64bit
-if(USE_FBGEMM AND NOT CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64")
+if(USE_FBGEMM AND NOT CMAKE_SYSTEM_PROCESSOR MATCHES "^(x86_64|aarch64)$")
```

Comment:
Q: Do you want to extend it to Apple Silicon as well? In that case, one needs to add arm64 there (but perhaps could be done as part of separate PR)